lib/index.js

```diff
@@ -27,11 +27,14 @@ api.dereferenceElectorPool = async ({electorPool, ledgerNode}) => {
 
   for(const e of electorPool) {
     // service may be a string referencing a serviceId contained in a DID
-    // Document referenced in `e.elector`
     if(typeof e.service === 'string') {
+      // extract the DID from `service` which is a DID followed by a fragment
+      const recordId = e.service.substr(0, e.service.indexOf('#'));
```

Comment:
As long as a validator ensures the `#` is there, we're good here.

Reply:
right, currently being validated as a string, I'll add an issue in validator.